# 15 ms-invert
# defect: middle/side is chosen when its perceptual entropy is the larger
# caught by: Mp3EncodeStereo.ACorrelatedPairIsCodedAsMiddleAndSide
#
--- a/src/codec/mp3/mp3enc_encoder.c
+++ b/src/codec/mp3/mp3enc_encoder.c
@@ -333,7 +333,7 @@
       ms += gaud_mp3e_estimate_bits(layout, mid_side[gr][0], ms_allowed[gr][0]);
       ms += gaud_mp3e_estimate_bits(layout, mid_side[gr][1], ms_allowed[gr][1]);
     }
-    middle_side = ms < lr;
+    middle_side = ms > lr;
     if (middle_side) {
       for (unsigned gr = 0; gr < granules; ++gr) {
         const MP3E_Layout * layout = &enc->layout[layout_index[gr]];
